Extending/UI-Library/index.md

```diff
@@ -6,24 +6,22 @@ meta.Description: "A guide for getting started working with the Umbraco UI Libra
 # UI Library
 
 :::note
-Please be aware that the topic covered in this document is not a part of the Umbraco CMS at this time.
+Please be aware that the topic covered in this document is currently only available in the Release Candidate (RC) of Umbraco 10.3.
 
-In the [Backoffice UI API Documentation](../Backoffice-UI-API-Documentation/) article you can find links to relevant resources for working with the Umbraco backoffice.
+In the [Backoffice UI API Documentation](../Backoffice-UI-API-Documentation/) article you can find links to relevant resources for working with the Umbraco backoffice in previous versions.
```

Comment:
The API documentation is still relevant until the new Backoffice is released. The UI Library is opt-in and a way for package developers to start migrating beforehand.